buildshiprun/handler.go

```diff
@@ -297,6 +298,7 @@ func getEvent() (*sdk.Event, error) {
 	info.URL = os.Getenv("Http_Url")
 	info.Image = os.Getenv("Http_Image")
 	info.SCM = os.Getenv("Http_Scm")
+	info.RepoURL = os.Getenv("Http_Repositoryurl")
```

Comment:
This looks pretty odd compared to the other strings. What if we separate it with another dash?

Reply:
Also we use the word Repo elsewhere, we could do that too? `Http_Repo_Url` (or whatever the watchdog converts it to)?

Reply:
`Repo_URL` is transformed to `Http_Repo_url`